swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    
+    def __init__(self, category="Clothing", condition=0, age=None):
+        super().__init__(category, condition, age)
```

Comment:
Great implementation of inheritance in here and `Decor` and `Electronics`! Since `category` will always be set as `"Clothing"`, we can even pull it out of our Clothing child class's initializer and  make our super initializer set it instead:
```
def __init__(self, condition=0, age=None):
    super().__init__(category="Clothing", condition, age)
```